### ISSUE TITLE
Boneattachment Does Not Work on First Bone in Skeleton

### ISSUE THREAD

**Issue by ngreve** [godotengine/godot#50053]
### Godot version

v3.3.3.rc.custom_build [64cfd5cba]

### System information

Arch Linux 

### Issue description

Using the `BoneAttachment` on the first bone of a skeleton, does not follow the animation. The model was created in and exported from Blender.

(The example project and this report mentions Godot 3.3.3, but I encountered this bug in 4.0, too.)

### Steps to reproduce

https://user-images.githubusercontent.com/18660100/124170380-a35b5f00-daa7-11eb-9d19-df688c99f38c.mp4

The video shows the model which was exported from Blender.
The model consists of to skeletons:
  - The red box contains one skeleton which consists of one single bone. This single bone has the `BoneAttachment` node attached to it, which in turn has the red prism as its child. The animation translates the bone in the up/down direction. The attached Prism does not follow.
  - The blue and green box do contain another skeleton which consists of two bones. The parent/root bone is attached to the blue box. A child bone is attached to the green box. Each of the bones do have a `BoneAttachment` node attached to them with the corresponding blue and green prisms. When the animation is played only the blue green prism, the prism which is attached to the child bone, follows the animation.

### Minimal reproduction project

[blender_bone_test.zip](https://github.com/godotengine/godot/files/6750030/blender_bone_test.zip)
[godot_bone_attachment_bug.zip](https://github.com/godotengine/godot/files/6750032/godot_bone_attachment_bug.zip)

**Comment by Calinou** [comment]
Can you reproduce this after switching to the GLES2 backend in the top-right corner in the editor? Can you still reproduce it after enabling **Force Software Skinning** in the Project Settings?

**Comment by ngreve** [comment]
Yes, the bug occurs in every possible combination of the suggested settings.

**Comment by fire** [comment]
I'll need to look at it, but I think a singular bone isn't posable. Can you try posing the bone on a skinned mesh?

**Comment by ngreve** [comment]
> Can you try posing the bone on a skinned mesh?`

Sorry, I really don't know what you mean by that.


> I think a singular bone isn't posable

The project shows that the single bone skeleton is very likely not the root cause of the problem; it seems that explicitly the parent or first bone in a chain of bones causes the problem. If a single-bone-skeleton would be the problem, then the blue prism should follow the blue box, if I am not mistaken.
